pkg/kubernetes/annotation.go

```diff
@@ -65,6 +65,22 @@ func (c *controller) AddSpinnakerAnnotations(u *unstructured.Unstructured, appli
 		}
 	}
 
+	if strings.EqualFold(gvk.Kind, "daemonset") {
+		ds := NewReplicaSet(u.Object)
+
+		// Add spinnaker annotations to the replicaset pod template.
```

Comment:
Change this comment to reference "daemonSet" instead of "replicaset".

Reply:
Line 69 looks pretty important to change as well